src/api/protectedApiClient.ts

```diff
@@ -13,13 +14,16 @@ export interface ProtectedApiClient {
   readonly getRequestStatuses: () => Promise<PersonalRequest[]>;
   readonly makePFRequest: () => Promise<void>;
   readonly deactivateAccount: () => Promise<void>;
+  readonly getContactInfo: () => Promise<ContactInfo>;
+  readonly setContactInfo: (request: ContactInfo) => Promise<void>;
 }
 
 export enum ProtectedApiClientRoutes {
   CHANGE_PASSWORD = '/api/v1/protected/user/change_password',
   REQUEST_STATUSES = '/api/v1/protected/requests/status',
   MAKE_PF_REQUEST = 'api/v1/protected/requests',
   USER = '/api/v1/protected/user',
+  CONTACT_INFO = '/api/v1/protected/user/contact_info',
```

Comment:
⛏ 
```
const BASE = '/api/v1/protected';
export enum ProtectedApiClientRoutes {
   CHANGE_PASSWORD = `${BASE}/user/change_password`,
   REQUEST_STATUSES = `${BASE}/requests/status`,
   ....
}
```
Removes a teeny tiny bit of risk of mistyping the route. Totally optional tho

Reply:
might as well.